stdlib/public/core/Sequence.swift

```diff
@@ -1067,7 +1070,10 @@ extension Sequence {
       } else {
         result.append(ringBuffer[i])
         ringBuffer[i] = element
-        i = (i + 1) % k
+        i += 1
+        if i == k {
```

Comment:
We should be consistent in how we manipulate indices; above you use `0`, here you use `startIndex`, but also `k`, which would be more consistent with `0`.

Reply:
Ok I updated it!